### PR TITLE
Wait for install on async service creation

### PR DESCRIPTION
**Pull request by f0rmiga**
## Description

- Refactored to use the official helm package.
- Updated helm tiller and package from 2.8.1 to 2.13.1.
- Dropped the tiller package due to not being used anymore.
- Refactored the make targets for handling better local testing.
- Refactored the minibroker package to handle passing options to helm on installing and added `helm.InstallWait(true)` when doing async provisioning.

## Test plan

Run all tests:

```
$ minikube start
$ eval $(minikube docker-env)
$ make create-cluster image deploy
$ make test
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,8 +1,8 @@
 REPO ?= github.com/osbkit/minibroker
 BINARY ?= minibroker
 PKG ?= $(REPO)/cmd/$(BINARY)
-REGISTRY ?= osbkit/
-IMAGE ?= $(REGISTRY)minibroker
+REGISTRY ?= splatform/
```
Comment (andreas-kupries) [inline Makefile]: Will using this suse-specific value make acceptance upstream more difficult ?

Comment (f0rmiga) [reply Makefile]: Most likely, yes. At the same time, we have this already https://github.com/SUSE/minibroker/blob/suse-cf/charts/minibroker/values.yaml#L26.

Comment (mook-as) [reply Makefile]: You should probably base your branch on the _upstream_ branch, and then modify it for us in a PR to merge the upstream changes?

**Comment by f0rmiga** [comment]
Closing in favor of https://github.com/SUSE/minibroker/pull/13.

**Pull request closed without merging**